src/collective/contact/widget/templates/contact_input.pt

```diff
@@ -3,6 +3,16 @@
              xmlns:metal="http://xml.zope.org/namespaces/metal"
              xmlns:i18n="http://xml.zope.org/namespaces/i18n">
     <div tal:attributes="id string:${view/id}-autocomplete">
+        <div tal:define="voc view/field/prefilter_vocubulary"
+             tal:condition="voc">
+            <label for="prefilter-select" i18n:translate="">Choose a filter:</label>
+            <select name="prefilters" id="prefilter-select">
+                <option value="" i18n:translate="">No filter</option>
```

Comment:
It's preferable to manage a "Choose value" in the vocabulary because in ia.docs case, the vocabulary will often contain one value for the current user. I will manage this in the vocabulary.

Reply:
If I understand well:

- remove the _label_ tag ("choose a filter")
- remove the first _option_ tag ("no filter")

> It's also needed to manage a default value method.

- that method is set as an attribute of _ContactList_, like _prefilter_vocubulary_.
- does it need to take parameters ?
- the method returns a vocabulary value. when rendering the select options, "selected=selected" is added to the matching option.

Reply:
No need to manage a parameter, just context aware.